frontend/src/app/main/ui/workspace/sidebar/options/shapes/frame.cljs

```diff
@@ -71,58 +71,59 @@
         variants?                 (features/use-feature "variants/v1")
         is-variant?               (when variants? (:is-variant-container shape))]
 
-    (if is-variant?
-      [:> variant-menu* {:shapes [shape]}]
-      [:*
-       [:& layer-menu {:ids ids
-                       :type shape-type
-                       :values layer-values}]
-       [:> measures-menu* {:ids ids
-                           :values measure-values
-                           :type shape-type
-                           :shape shape}]
-
-       [:& component-menu {:shapes [shape]}]
-
-       [:& layout-container-menu
-        {:type shape-type
-         :ids [(:id shape)]
-         :values layout-container-values
-         :multiple false}]
-
-       (when (and (= (count ids) 1) is-layout-child? is-grid-parent?)
-         [:& grid-cell/options
-          {:shape (first parents)
-           :cell (ctl/get-cell-by-shape-id (first parents) (first ids))}])
-
-       (when (or is-layout-child? is-layout-container?)
-         [:& layout-item-menu
-          {:ids ids
-           :type shape-type
-           :values layout-item-values
-           :is-flex-parent? is-flex-parent?
-           :is-grid-parent? is-grid-parent?
-           :is-flex-layout? is-flex-layout?
-           :is-grid-layout? is-grid-layout?
-           :is-layout-child? is-layout-child?
-           :is-layout-container? is-layout-container?
-           :shape shape}])
-
-       (when (or (not ^boolean is-layout-child?) ^boolean is-layout-child-absolute?)
-         [:& constraints-menu {:ids ids
-                               :values constraint-values}])
-
-       [:& fill-menu {:ids ids
+    [:*
+     [:& layer-menu {:ids ids
+                     :type shape-type
+                     :values layer-values}]
+     [:> measures-menu* {:ids ids
+                         :values measure-values
+                         :type shape-type
+                         :shape shape}]
+
+     [:& component-menu {:shapes [shape]}]
+
+     (when is-variant?
+       [:> variant-menu* {:shapes [shape]}])
```

Comment:
I think we should reuse the `[shape]`, avoiding creating a new vector with a single shape on each rerender, twice.